docs/pages/inboxes/push-notifs/understand-push-notifs.md

```diff
@@ -0,0 +1,140 @@
+
+### How DM stitching works
+
+1. When fetching messages from any of the MLS groups associated with a DM conversation, the XMTP SDK responds with messages from all of the groups associated with the DM conversation.
+    - For example, let’s say you have three MLS groups associated with a DM conversation:
+        - alix-bo-1
+        - alix-bo-2
+        - alix-bo-3
+        
+        Any messages sent in any of these DM conversations will display in all of these DM conversations.
+        
+        For example, `alix.eth` sends a message in alix-bo-1. `bo.eth` is on alix-bo-3, but can still see messages sent in alix-bo-1.
+        
+2. When sending messages in a DM conversation, all installations in that DM will eventually converge to sending them to the same MLS group, even if they originally start off using different ones.
+    - For example, `bo.eth` sends a message in alix-bo-3. `alix.eth` is on alix-bo-1, but can still see messages from alix-bo-3. When `alix.eth` sends a reply to `bo.eth`, it uses the most recently used DM conversation, alix-bo-3. In this way, all messaging will eventually move to alix-bo-3, and 1 and 2 will slowly fade away due to non-use.
+
+### DM stitching considerations for push notifications
+
+DM stitching addresses provides a unified UX in the app. However, the multiple DM conversations under the hood must still be addressed for push notifications.
+
+Let’s take DM conversations alix-bo-1 and alix-bo-3 between `alix.eth` and `bo.eth`. With DM stitching, these two conversations display as one conversation. However, we must remember that they have two different conversation IDs, and thus two different topics. 
+
+Therefore, when subscribing a DM conversations to push notifications, you should subscribe to a list of topics because every DM conversation can potentially have multiple topics. You will miss push notifications for messages if you are not listening to every potential topic for a DM conversation that messages could potentially be sent on.
+
+For example, you must consider that with DM stitching, the conversation is moving from alix-bo-1 and alix-bo-3, for example, and [resubscribe appropriately](/inboxes/push-notifs/push-notifs#resubscribe-to-topics-to-get-new-hmac-keys). 
+
+Also, you must consider that a welcome message will be sent when a DM conversation is added to the stitched DMs, and you should not send a push for the welcome message because the user already has a conversation with the person. It is just a different DM conversation in the set of DM conversations that are stitched together. These welcome messages are filtered out of streams, but they are not filtered out for the XMTP push notification server, so you must handle these duplicate DM welcomes at the push notification service. This is only an issue if you are building an iOS app without the user notification entitlement, as there are no HMAC keys associated with this new welcome.
```

Comment:
```suggestion
Also, you must consider that a welcome message will be sent when a DM conversation is added to the stitched DMs, and you should not send a push for the welcome message because the user already has a conversation with the person. It is just a different DM conversation in the set of DM conversations that are stitched together. These welcome messages are filtered out of streams, but they are not filtered out for the XMTP push notification server, so you must handle these duplicate DM welcomes at the push notification service.
```